tests/gauss_newton_test.py

```diff
@@ -83,7 +83,7 @@ def test_enzyme_reaction_parameter_fit(self):
                               onp.array([0.36183689, 0.55626653]))
 
 
-  @parameterized.product(implicit_diff=[True, False])
+  @parameterized.product(implicit_diff=[False, False])
```

Comment:
I haven't forgot this :). It is intentionally set as False as well, cause `True` doesn't pass the test. I basically tried what I did for Gauss-Newton, while the example here has many more data points (the number of x and y values are 40+). It would be great if you can take a look and see if I am using the implicit differentiation correctly. Thanks.

Reply:
But you didn't change anything in `gauss_newton.py` so why should the tests stop working?

Reply:
Oh my bad. I meant to avoid implicit differentiation in the LM file and I must have looked at the wrong file when making the change. Thank you for pointing that out.